src/ccontrol/UserQueryFactory.cc

```diff
@@ -365,6 +366,9 @@ UserQuery::Ptr UserQueryFactory::newUserQuery(std::string const& aQuery, std::st
             LOGS(_log, LOG_LVL_INFO,
                  "Set SELECT COUNT(*) row count optimization to "
                          << (_useQservRowCounterOptimization ? "ON" : "OFF"));
+        } else if (setQuery->varName() == "QSERV_DEBUG_CZAR_NO_MERGE") {
+            _debugNoMerge = setQuery->varValue() != "0";
+            LOGS(_log, LOG_LVL_INFO, "SET QSERV_DEBUG_CZAR_NO_MERGE " << (_debugNoMerge ? "1" : "0"));
```

Comment:
Very good point! I will change the level as you suggested.

And yes, SQL interface is not the best way of modifying Qserv behavior. Unfortunately, that's the only one we have that allows dynamic change. Another option would be to put a flag into `czar`'s config file. However, the latter would require restarting `czar` ad it will also complicate tests.